hudi-utilities/src/test/java/org/apache/hudi/utilities/sources/helpers/TestKafkaOffsetGen.java

```diff
@@ -140,11 +140,13 @@ public void testGetNextOffsetRangesFromMultiplePartitions() {
     testUtils.sendMessages(testTopicName, Helpers.jsonifyRecords(dataGenerator.generateInserts("000", 1000)));
     KafkaOffsetGen kafkaOffsetGen = new KafkaOffsetGen(getConsumerConfigs("earliest", "string"));
     OffsetRange[] nextOffsetRanges = kafkaOffsetGen.getNextOffsetRanges(Option.empty(), 499, metrics);
-    assertEquals(2, nextOffsetRanges.length);
+    assertEquals(3, nextOffsetRanges.length);
```

Comment:
why should the expected value change?

Reply:
The eventsPerPartition for 499 messages and 2 partitions is 249 messages per offsetRange, To fill up 499 messages 3(249 + 249 + 1) offset ranges.